sensors/MagnetSensor/JsonElement.cpp

```diff
@@ -0,0 +1,725 @@
+#include "JsonElement.h"
```

Comment:
Instead of copying ```JsonElement.cpp``` and ```JsonElement.h``` we can add a package.json file with:

 ```{
  "name": "arduino-2357",
  "version": "0.1.0",
  "description": "FRC 2357 Arduino libraries and projects",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "repository": {
    "type": "git",
    "url": "git+https://github.com/frc2357/arduino-2357.git"
  },
  "author": "System Meltdown - FRC 2357",
  "license": "ISC",
  "bugs": {
    "url": "https://github.com/frc2357/arduino-2357/issues"
  },
  "homepage": "https://github.com/frc2357/arduino-2357#readme",
  "arduinoLibs": [
    "../libraries/JsonState"
  ]
}
```
Then install npm and you should be able to run ```npm install yourPath/package.json```. This will download the Json Element stuff to your actual Arduino library.
What you currently have works, but having each project rely on the same files makes updating our JSON stuff easy.